Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -659,29 +659,20 @@ private void RenderNodes (HtmlTextWriter writer, WebTreeNodeCollection webTreeNo
         }
 
         writer.AddAttribute(HtmlTextWriterAttribute.Id, "Node_" + nodeID);
-        writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.TreeItem);
-        if (hasChildren || !node.IsEvaluated)
-        {
-          writer.AddAttribute(
-              HtmlTextWriterAttribute2.AriaExpanded,
-              node.IsExpanded ? HtmlAriaExpandedAttributeValue.True : HtmlAriaExpandedAttributeValue.False);
-        }
-        if (node.IsSelected)
-        {
-          writer.AddAttribute(HtmlTextWriterAttribute2.AriaSelected, HtmlAriaSelectedAttributeValue.True);
-        }
-        writer.AddAttribute("tabindex", _focusededNode == node ? "0" : "-1");
+
         if (EnableTopLevelGrouping && isTopLevel && !isLastNode && nodes[i].Category != nodes[i + 1].Category)
           writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassNodeCategorySeparator);
 
+        writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.None);
```

Comment:
it seems this change/commit should be part of `RM-9152 change tree view to improve support for wcag` because it always had a role. it just got lost inside this feature branch and then re-added as `none` but in a separate comment.